openslides/motions/views.py

```diff
@@ -893,6 +895,54 @@ def check_view_permissions(self):
             result = False
         return result
 
+    def create(self, *args, **kwargs):
+        try:
+            response = super().create(*args, **kwargs)
+        except WorkflowError as e:
```

Comment:
Which WorkflowError do you expect here?

Reply:
You currently can provide the `first_state_id` and the serializer will set it. This just can result in a WorkflowError, because if the state exists (checked by the serializer) it will be in the wrong workflow.
Dou you have an idea how to prevent this. Shall we just delete the key `first_state_id` here?

Reply:
I propose to set the first_state field in the serializer to read_only=True. Then we do not have this problems and the flow to create new workflows is clear.

Reply:
THis just works, if we forbid to set the first state. Emanuel and I had a discussion about this and we may think, we should leave the first_state fixed, so no deletion of it and no change of the relationship. One can just change all attributes.
But we want this to be merged. I'll create another issue, because removing this functionality should be easy, but needs testing and I do not have the time today.